schema.sql

```diff
@@ -1,5 +1,5 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE DATABASE vet_clinic ;
+
+CREATE TABLE animals(id INT, name TEXT, date_of_birth DATE, escape_attempts INTEGER, neutered BOOLEAN, weight_kg DECIMAL);
```

Comment:
- [ ] (Optional) It would be nice to auto-generate the ID instead of manually inserting the value in the data.sql. To auto-generate the `id`, you can do any of the following:

`id SERIAL PRIMARY KEY` OR

`id INT GENERATED ALWAYS AS IDENTITY`

Reply:
Thank you @Gambit142 for pointing out these excellent suggestions. I have implemented it. :handshake:

Reply:
@levy002, you are welcome, I am glad I could help out. 🤝